### ISSUE TITLE
[CAIA Intake] DBEX-TREX: Timely and Truthful Submission Experience

### ISSUE THREAD

**Issue by mayacarrolluxa6** [department-of-veterans-affairs/va.gov-team#92471]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

We have passed the midpoint review and have conducted evaluative research on our proposed designs. Also we have reviewed internally with our team's a11y specialist. Based on [our findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-07-Submission-Status/Research-findings.md), we have made revisions and are seeking feedback primarily on content, and any major issues you see in terms of accessibility and information architecture. While the ask is still broad, we have identified specific areas that would benefit from your expertise in refining some rough content, captured in [Figma comments](https://www.figma.com/design/qyTtKDOTHZPGU59PAfqVhq/Submission-Experience?node-id=5626-129729&t=w4ISChxpeifZmQDY-1), and which we can discuss further during our meeting.

## About your team

Team name: Disability Benefits
OCTO product owner: @emilytheis Emily Theis
Product name: 526 Disability Benefits Application
Product manager: @pacerwow Jared Pace
Slack channel: [benefits-disability-design](https://dsva.slack.com/archives/C053UDWMH7U)
Dedicated content writer on your team (if you have one): none
Dedicated a11y specialist on your team (if you have one): @bacitracin Tracy Tran
 
## About your initiative
Many Veterans do not have a positive experience when submitting a claim for disability compensation. There are multiple issues that describe errors, confusion, a lack of clarity, missing information, and cognitive load that isn't conducive to the level of quality we aspire to provide for Veterans. This feature focuses on improving the submission experience starting at the point when the Veteran clicks "submit" on a disability benefits application on va.gov and ending when the claim is established in VBMS and all initial doc uploads succeed or the submission process ultimately fails.

More in [our feature brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/product/feature-briefs/Submission%20Experience.md).

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [X] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

We are redesigning touchpoints with Veterans including screens and emails that happen after a Veteran clicks "submit" on a disability benefits application. We are considering success, backup path, and multiple failure scenarios.

We are / will be coordinating with other teams on this, notably the Veteran Facing Forms team, My VA / Authenticated Experience, and Benefits Management Tools.

Success looks like:

- Ensure the expectations of the claim submission timeline are set truthfully for the Veteran.
- Ensure the Veteran understands (at a basic level) what is happening with their claim submission. 
- Ensure the language we use is straightforward so that Veterans feel understanding submission status is trivial.
- Ensure an email confirmation is received in a timely manner and contains information the Veteran deems useful.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [X] No
- [ ] I’m not sure

### Where are you at in your timeline?
We worked with CAIA earlier in the collaboration cycle (https://github.com/department-of-veterans-affairs/va.gov-team/issues/83554) and are requesting to partner with CAIA again for an additional round of copy review.

We are preparing to hand off to our engineering teams for development in an upcoming sprint.

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [X]  Yes
- [ ]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- Unknown
- **What is the name of your engineer who will build the React widget?** @christinec-fftc Christine Cereca

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [X] Design intent 
- [X] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide a link to your Collaboration Cycle Request:
- N/A 

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/product/feature-briefs/Submission%20Experience.md)
- [Timely and Truthful Epic](https://app.zenhub.com/workspaces/disability-benefits-experience-team-1-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/82076)
- [Figma designs](https://www.figma.com/design/qyTtKDOTHZPGU59PAfqVhq/Submission-Experience?node-id=5626-129729&t=iMjBr9ubAHdeyj7n-1) with comments for CAIA
- [Figma Prototype ](https://www.figma.com/proto/qyTtKDOTHZPGU59PAfqVhq/Submission-Experience?page-id=5626%3A129729&node-id=5626-130341&node-type=FRAME&viewport=-1166%2C-8928%2C0.5&t=nPBreO4YLEmPjYYl-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=5626%3A130341&show-proto-sidebar=1)
- [Research Findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-07-Submission-Status/Research-findings.md)

## Next steps
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92584
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92617
```

**Comment by mayacarrolluxa6** [comment]
@mengA6 Looping in Mike for visibility.

**Comment by laurwill** [comment]
Hi @mayacarrolluxa6 , thanks for looping us back in. I see you're getting ready to hand off to engineering in an upcoming sprint—we'd definitely like to work on this with you before that handoff, since it connects up to so much other post-submission work across products. I just requested access to your Google cal—I'll try to find a time to sync on this next week!

**Comment by mayacarrolluxa6** [comment]
Hi @laurwill nice to meet you. I've granted you access to my calendar and I look forward to partnering with you on this project.

**Comment by mayacarrolluxa6** [comment]
@laurwill I've sent you a meeting request to sync tomorrow, please let me know if the proposed time works with your schedule. Thanks!

**Comment by coforma-jamie** [comment]
@mayacarrolluxa6 we left a comment for you [here](https://www.figma.com/design/qyTtKDOTHZPGU59PAfqVhq?node-id=7079-295842#952563276) :)

**Comment by mayacarrolluxa6** [comment]
@laurwill @coforma-jamie @EvanAtCoforma 

Hi Laura, Jamie, and Evan,

Thank you for your time today! I appreciate the valuable feedback and insights you shared.

Could you please provide an estimate of how long you anticipate the CAIA review will take to complete? Also, how will I be notified once your review is finished?

**Comment by laurwill** [comment]
Hey @mayacarrolluxa6 , just documenting here that we chatted over slack, and I added a bunch of content-related comments to the Figma file — let me know if a sync would help to discuss any of those!

**Comment by mayacarrolluxa6** [comment]
Hi @laurwill our team is almost done reviewing and implementing all of the excellent feedback and suggestions. I don't think we have any follow up questions at this point, do you want us to close this issue?

**Comment by laurwill** [comment]
Hi @mayacarrolluxa6 and team, yes we can close this for now — just documenting that per slack conversations, your team planned to implement some initial updates to the confirmation page, and you'll come back to request another round of CAIA collaboration on the confirmation page as you adjust how your email notifications work. 

And noting a couple of items we discussed in our most recent sync:
- It's important to keep the confirmation page aligned with the current email notifications you're sending, so that the messaging about what to expect is accurate and consistent. 
- As your team works to implement the full [stay informed of submission status pattern](https://design.va.gov/patterns/help-users-to/stay-informed-of-their-application-status), you'll need to work with the team that owns the claim status tool to make sure that your confirmation page, your emails, and your submission status cards in My VA are all consistent with and are not duplicating notifications or status updates provided by or reflected in the claim status tool. 
     - For example, at the moment you send an email that says "We received your disability claim," I'd expect that claim to show as "Claim received" in the CST. But I'm not sure how the cards in My VA would work — would this single claim show up twice, once as a "Received" card in the Benefit applications and forms section, and once as a "Claim received" card in the Claims and appeals section? Those are the types of questions that we'll need to resolve, if you haven't already figured it out with the CST team.

Let me know if you have any questions, and looking forward to collaborating on future updates with you all.
